test/test_parabolic_3d.jl

```diff
@@ -0,0 +1,88 @@
+module TestExamplesParabolic3D
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+EXAMPLES_DIR = joinpath(examples_dir(), "dgmulti_3d")
```

Comment:
```suggestion
```
You do not use this, do you?

Reply:
No, it doesn't use this folder at all. But I vaguely recall that removing this lines caused these tests to not run properly so I just left it there.

Reply:
Would you mind trying that again? If it gives problems, it's an indication that I messed up our testing macros and should fix them :sweat_smile:

Reply:
Okay, the tests do run locally for me if I remove this unused path information. It just throws a warning at the beginning.